.github/workflows/v1.deployDotNetCoreAppToAzureWebApp.yml

```diff
@@ -1,5 +1,11 @@
+name: V1_BuildAndDeploy_AspNetCoreApp_ToAzureWebApp
+
 on:
   push:
+    paths:
+      - sampleApps/dotNetCore/**
```

Comment:
If we plan on supporting multiple samples per platform in the future, I think we should scope the paths within a workflow to the specific platform samples they are using -- in this case, we'd scope it down to `sampleApps/dotNetCore/NetCoreApp31.MvcApp/**`.